utility/make_private_namespace.py

```diff
@@ -0,0 +1,45 @@
+""" Create a namespace """
+import os
+import subprocess
+import shlex
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def make_private_namespace(name, region='us-east-2'):
+    """
+    Create a namespace.
+
+    Args:
+        name (str): namespace to be created
+        region (str): aws region.  Defaults to us-east-2
+
+    Returns:
```

Comment:
I don't understand this test,
It will never fail

Reply:
I have changed this routine and the comment to return False on failures.